### PR TITLE
EventManager

### PR DESCRIPTION
**Pull request by nspruit**
Refs #2

### CODE REVIEW

File: WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/EventInitializationManager.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+		eventsToTransferPersister = new EventPersisterBase(
+				toTransferDatabaseFile);
+		eventsStatisticsPersister = new EventPersisterBase(
+				statisticsDatabaseFile);
+		eventManager = new EventManager(eventsToTransferPersister,
+				eventsStatisticsPersister);
+		eventManager.setSessionSeed(IntervalInitializationManager.getInstance()
+				.getIntervalManager().getSessionSeed());
+
+		// TODO: init listeners and transfer manager
```
Comment (Inventitech) [inline WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/EventInitializationManager.java:39]: Left-over TODO?

File: WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/EventInitializationManager.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        File toTransferDatabaseFile = new File(baseFolder, WatchDogUtils.getProjectName() + "events.mapdb");
+        File statisticsDatabaseFile = new File(baseFolder, WatchDogUtils.getProjectName() + "eventsStatistics.mapdb");
+
+        eventsToTransferPersister = new EventPersister(toTransferDatabaseFile);
+        eventsStatisticsPersister = new EventPersister(statisticsDatabaseFile);
+
+        eventManager = new EventManager(eventsToTransferPersister, eventsStatisticsPersister);
+        eventManager.setSessionSeed(sessionSeed);
+
+        //TODO: init listeners and transfer manager
```
Comment (Inventitech) [inline WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/EventInitializationManager.java:45]: left-over TODO?

File: WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/EventInitializationManager.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    /**
+     * Closes the database. The database can recover even if it is not closed
+     * properly, but it is good practice to close it anyway.
+     */
+    public void shutdown(String projectName) {
+        eventsToTransferPersister.closeDatabase();
+        eventsStatisticsPersister.closeDatabase();
+        instances.remove(projectName);
+        //TODO: remove listeners
```
Comment (Inventitech) [inline WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/EventInitializationManager.java:70]: left-over TODO?

File: WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/EventInitializationManager.java (hunk 1 above)
Comment (nspruit) [inline WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/EventInitializationManager.java:39]: TODO is done and removed in later PR

File: WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/EventInitializationManager.java (hunk 2 above)
Comment (nspruit) [inline WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/EventInitializationManager.java:45]: TODO is done and removed in later PR

File: WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/EventInitializationManager.java (hunk 3 above)
Comment (nspruit) [inline WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/EventInitializationManager.java:70]: This one was actually left-over and is now removed, as this doesn't have to be done.

**Pull request merged**